examples/pe_example.py

```diff
@@ -0,0 +1,47 @@
+import torch
+import eigenpro.kernels as kernels
+import eigenpro.models.sharded_kernel_machine as skm
+import eigenpro.solver as solver
+import eigenpro.utils.device as dev
+from datasets import load_dataset
+
+n, p, d, c = 1000, 100, 5, 2
```

Comment:
what does n, p, d, c stand for?
could you add a comment to explain or include a reference?

Reply:
I added these